src/main/resources/configCivcraft.yml

```diff
@@ -7216,7 +7216,7 @@ recipes:
         durability: 3
       eggs:
         material: EGG
-        amount: 8
+        amount: 4
```

Comment:
I'm not opposed to removing them or replacing them. As long as each of the recipes takes about the same amount of effort to run.